src/lib/select/select.ts

```diff
@@ -461,6 +464,12 @@ export class MatSelect extends _MatSelectMixinBase implements AfterContentInit,
     }
   }
 
+  ngOnChanges(changes: SimpleChanges) {
+    if (changes.disabled) {
```

Comment:
Did these changes make it through as a part of another branch?

Reply:
This was fixing a bug I noticed when making one of the demos

Reply:
Add a comment here?